po/POTFILES.skip

```diff
@@ -0,0 +1,13 @@
+data/10_mate-wayland.gschema.override
+data/firedecor.config
+data/mate/close.png
+data/mate/close-hovered.png
+data/mate/close-pressed.png
+data/mate/minimize.png
+data/mate/minimize-hovered.png
+data/mate/minimize-pressed.png
+data/mate/toggle-maximize.png
+data/mate/toggle-maximize-hovered.png
+data/mate/toggle-maximize-pressed.png
```

Comment:
Ihmo, you don't need the graphic files. But check how it is handle in other repositories.